factcast-itests/factcast-itests-factus/src/test/java/org/factcast/itests/factus/client/FactusClientTest.java

```diff
@@ -886,6 +887,28 @@ private static void sleep(long ms) {
     Thread.sleep(ms);
   }
 
+  @Nested
+  class Clock {
+
+    @Test
+    void currentTimeInMillis() {
+      long beforeMillis = System.currentTimeMillis();
+      long now = factus.currentTimeInMillis();
+      long afterMillis = System.currentTimeMillis();
+
+      assertThat(now).isGreaterThanOrEqualTo(beforeMillis).isLessThanOrEqualTo(afterMillis);
+    }
+
+    @Test
+    void currentTimeAsInstant() {
+      Instant before = Instant.now();
```

Comment:
bit flaky, isn't it? This relies on local clock and PG clock being reasonably in sync.

Reply:
Sorry for the late response, I think I need to turn on github notifications again :D 

I don't think so, in this test we use the pg from a docker container, and AFAIK their time would be the same as the host that is running the tests.